unittests/test_schedulers.py

```diff
@@ -158,13 +153,7 @@ def _expected_slurm_directives(job):
         '#SBATCH --ntasks-per-socket=%s' % job.num_tasks_per_socket,
         '#SBATCH --cpus-per-task=%s' % job.num_cpus_per_task,
         '#SBATCH --hint=multithread',
-        '#SBATCH --nodelist=%s' % job.sched_nodelist,
-        '#SBATCH --exclude=%s' % job.sched_exclude_nodelist,
-        '#SBATCH --partition=%s' % job.sched_partition,
-        '#SBATCH --reservation=%s' % job.sched_reservation,
-        '#SBATCH --account=%s' % job.sched_account,
```

Comment:
If you pass them as job options when creating the job you will have to hardcode them here, but it doesn't look like a problem to me.